src/test/scala/com/weightwatchers/reactive/kinesis/consumer/ConsumerProcessingManagerSpec.scala

```diff
@@ -118,12 +103,12 @@ class ConsumerProcessingManagerSpec
 
         //validate the probe received the Seq of ConsumerEvents
         val expectedMsg = ProcessEvents(
-          Seq(toConsumerEvent(record1),
-              toConsumerEvent(record2),
-              toConsumerEvent(record3),
-              toConsumerEvent(record4),
-              toConsumerEvent(record5),
-              toConsumerEvent(record6)),
+          ArrayBuffer(toConsumerEvent(record1),
```

Comment:
why `ArrayBuffer`?

Reply:
Mmmh no reason I remember. Revert to Seq.